### PR TITLE
1st submission

### PR DESCRIPTION
**Pull request by efeng732**
1/11 mini challenge

### CODE REVIEW

**Comment by efeng732** [comment]
Final change

File: index.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
 // ***** Question 1 *****
+let counter = 1;
+counter = 2;
+
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+ //console.log("%cQuestion 1", "color: red")
```
Comment (rlc900) [inline index.js:6]: 👍

File: index.js [hunk 2]
```diff
@@ -64,6 +91,13 @@
 
 
 // ***** Question 6 *****
+function printNameAndPhones(users) {
+  users.forEach(function (userObj){
+    console.log(userObj.name);
+    console.log(`Cell: ${userObj.phones.cell}`);
+    console.log(`Office: ${userObj.phones.office}`);
+  })
+}
```
Comment (rlc900) [inline index.js:100]: Nice job!!

File: index.js [hunk 3]
```diff
@@ -139,6 +183,13 @@ const line = []
 
 // ***** Scope & Closures - Question 1 *****
 
+let ticketNumber = 0;
+function takeATicketNumber(line) {
+  ticketNumber += 1; 
+  line.push(ticketNumber);
+  return `Welcome. You are ticket number ${ticketNumber}`;
+
+}
```
Comment (rlc900) [inline index.js:192]: Yass

File: index.js [hunk 4]
```diff
@@ -161,6 +212,17 @@ const line = []
...
+  let ticketNum = 0;
+  function takeATicketNumber (line) {
+  ticketNum += 1; 
+  line.push(ticketNum);
+  console.log(`Welcome. You are ticket number ${ticketNum}`) ;
+
+  }
+  return takeATicketNumber;
+}
+
```
Comment (rlc900) [inline index.js:225]: This is great! You could also return the function itself on like 217 so 223 wouldnt be needed, but it's not incorrect!

**Review by rlc900 — COMMENTED** [review]
Really awesome job overall! And your code is so clean! 👍

**Pull request closed without merging**